domain-server/resources/web/web-new/package.json

```diff
@@ -16,10 +16,10 @@
         "axios": "^0.21.4",
         "core-js": "^3.6.5",
         "js-sha256": "^0.9.0",
+        "moment": "^2.29.4",
```

Comment:
Moment is deprecated. Is there a native solution we can use instead?

Reply:
Replaced uses of `moment` with the built-in JS `Date` class and uninstalled moment in commit [aff014f](https://github.com/vircadia/vircadia/pull/1728/commits/aff014fdc1d64d079b3a8687c352e756689416ae).